src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/issueadjournment/IssueAdjournmentNoticeAboutToStartHandlerTest.java

```diff
@@ -45,7 +46,7 @@ public class IssueAdjournmentNoticeAboutToStartHandlerTest {
 
     private SscsCaseData sscsCaseData;
 
-    @Before
+    @BeforeEach
     public void setUp() {
         openMocks(this);
```

Comment:
You can remove this now (and the tearDown method) and use (the better) annotation on the class:
@RunWith(SpringRunner.class)